modular_skyrat/master_files/code/game/machinery/hologram.dm

```diff
@@ -0,0 +1,14 @@
+GLOBAL_LIST_EMPTY(hologram_impersonators)
+
+/obj/machinery/holopad/set_holo(mob/living/user, obj/effect/overlay/holo_pad_hologram/h)
+	if(h.Impersonation)
+		GLOB.hologram_impersonators[user] = h
+		h.become_hearing_sensitive() // Well, we need to show up on "get_hearers_in_view()"
+	. = ..()
+
+/obj/machinery/holopad/clear_holo(mob/living/user)
+	var/obj/effect/overlay/holo_pad_hologram/h = GLOB.hologram_impersonators[user]
+	if(h)
+		h.lose_hearing_sensitivity()
+		GLOB.hologram_impersonators -= user
```

Comment:
Don't use `h`, please. Better name for this.

Also, if possible, rename the `Impersonation` variable to `impersonation` too, if that's a Skyrat variable and not a /tg/ one.

Reply:
It's a tg variable.